### PR TITLE
Integrate Metro 3.0.0-M3, includes JAXB 3.0.0-M5

### PR DESCRIPTION
**Pull request by lukasj**
this includes move to the new jakarta.ee namespace, see the change in quicklooks. Note that this may cause some regressions in (platform) TCKs. I prefer to fix them in the next integration (mostly to simplify debugging/testing of changes on my end) but I'm open to alternatives...

### CODE REVIEW

**Review by gurunrao — COMMENTED** [review]
There are more reference of old jaxb namespace based on search https://github.com/eclipse-ee4j/glassfish/search?p=1&q=%22http%3A%2F%2Fjava.sun.com%2Fxml%2Fns%2Fjaxb%22

Also on reference on old jaxws here - https://github.com/eclipse-ee4j/glassfish/search?q=%22http%3A%2F%2Fjava.sun.com%2Fxml%2Fns%2Fjaxws%22

**Comment by lukasj** [comment]
@gurunrao Are those tests being run regularly? If not, I think it's better to fix them in the next round or once one gets to clean those tests up. If yes, can you point me to the job running them?

How do I re-run the PR merge job? Failures there are far far away from what I see on my end

**Review by gurunrao — COMMENTED** [review]
PR-23207 Job for test run can be found here - https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/job/PR-23207/

Glassfish master test run can be found here - https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/job/master/

**Comment by lukasj** [comment]
@gurunrao none of those jobs is running devtests (those being unchanged thus having old ns).

Let me rephrase the question a bit -  how can one without committer status re-run the PR merge job?

**Comment by gurunrao** [comment]
> @gurunrao none of those jobs is running devtests (those being unchanged thus having old ns).
> 
> Let me rephrase the question a bit - how can one without committer status re-run the PR merge job?

Sure, we can revisit old jaxb, jaxws schema in devtests.
Let me know when PR merge job needs to be re-run, I will trigger the re-run.

**Comment by gurunrao** [comment]
@lukasj  job run can be found here - https://ci.eclipse.org/glassfish/job/glassfish_build-and-test-using-jenkinsfile/job/PR-23207/3/

**Review by gurunrao — APPROVED** [review]
LGTM

**Review by Cousjava — APPROVED** [review]

**Comment by lukasj** [comment]
@gurunrao these latest results - no failures - are the same as those on my end, thanks for the re-run!
